### PR TITLE
USS Marlow

### PR DESCRIPTION
**Pull request by Patternseeker**
# About the pull request

Adds USS Marlow

# Explain why it's good for the game

almayer yucky

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
add: Added USS Marlow.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -75,7 +75,6 @@
 	flags_gun_features = GUN_AUTO_EJECTOR|GUN_CAN_POINTBLANK|GUN_AMMO_COUNTER
 	starting_attachment_types = list(/obj/item/attachable/attached_gun/grenade, /obj/item/attachable/stock/rifle/collapsible)
 	map_specific_decoration = TRUE
-	start_automatic = TRUE
```
Comment (Kitsunemitsu) [inline code/modules/projectiles/guns/rifles.dm:78]: This is actually a balance change. Id' recommend atomizing this into a new pr.

File: code/modules/projectiles/guns/rifles.dm [hunk 2]
```diff
@@ -1566,14 +1565,15 @@
...
 /obj/item/weapon/gun/rifle/m4ra/set_gun_config_values()
 	..()
-	set_fire_delay(FIRE_DELAY_TIER_9)
-	set_burst_amount(0)
-	accuracy_mult = BASE_ACCURACY_MULT + HIT_ACCURACY_MULT_TIER_5
-	accuracy_mult_unwielded = BASE_ACCURACY_MULT - HIT_ACCURACY_MULT_TIER_4
-	damage_mult = BASE_BULLET_DAMAGE_MULT + BULLET_DAMAGE_MULT_TIER_8
-	recoil_unwielded = RECOIL_AMOUNT_TIER_4
-	damage_falloff_mult = 0
+	set_fire_delay(FIRE_DELAY_TIER_6)
```
Comment (Kitsunemitsu) [inline code/modules/projectiles/guns/rifles.dm:1568]: This is a mapping PR? Why are you buffing the M4RA.

File: code/modules/projectiles/guns/rifles.dm [hunk 3]
```diff
@@ -1566,14 +1565,15 @@
...
+	set_burst_amount(BURST_AMOUNT_TIER_2)
+	set_burst_delay(FIRE_DELAY_TIER_12)
+	accuracy_mult = BASE_ACCURACY_MULT + HIT_ACCURACY_MULT_TIER_2
 	scatter = SCATTER_AMOUNT_TIER_8
+	burst_scatter_mult = SCATTER_AMOUNT_TIER_8
```
Comment (Kitsunemitsu) [inline code/modules/projectiles/guns/rifles.dm:1573]: Are you adding burst too? This increases the damage of the M4RA significantly across long ranges. Do watch out for that.

File: strings/ai.txt [hunk 4]
```diff
@@ -104,7 +104,6 @@ SEN 5241
 SHODAN
 SID 6.7
 Setaur
-Shrike
```
Comment (Kitsunemitsu) [inline strings/ai.txt:107]: This is kinda entirely unrelated? 
I am in awe at how this got into this PR, and it shows a whole lack of care to this whole PR.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Changes requested. Please Fix before merge.

**Review by Ireuldiz — APPROVED** [review]
all is well- merging & making further edits upon this

**Pull request merged**